packages/merkle-distributor/scripts/example.json

```diff
@@ -0,0 +1,41 @@
+{
+  "chainId": 42,
+  "rewardToken": "0x47B1EE6d02af0AA5082C90Ea1c2c14c70399186c",
+  "windowIndex": 0,
+  "totalRewardsDistributed": "15000000000000000000",
+  "windowStart": 1614850539,
+  "recipients": {
+    "0x00b591bc2b682a0b30dd72bac9406bfa13e5d3cd": {
+      "amount": "1000000000000000000",
+      "metaData": { "reason": ["YD-WETH-21 Liquidity Mining Week 27"] }
+    },
+    "0x00e4846e2971bb2b29cec7c9efc8fa686ae21342": {
+      "amount": "2000000000000000000",
+      "metaData": { "reason": ["YD-WETH-21 Liquidity Mining Week 27", "Governance Votes KPI Options Round 1"] }
```

Comment:
it is not. the proof just consists of the hash of amount + wallet. it will change to include index in @nicholaspai latest PR.

It is simply included to enrich the front end.